Source/Parsers/SVGKParser.m

```diff
@@ -61,6 +61,8 @@ @implementation SVGKParser
 
 #define kThreadLocalCurrentlyActiveParser ( @"kThreadLocalCurrentlyActiveParser" )
 
+SVGKParser* getCurrentlyParsingParser(void);
```

Comment:
The prototype warning Xcode gives here seems to be incorrect, as all usage of this method happens after it is declared anyway. I looked about online and apparently just adding `void` to the empty parameter list of the definition of the method fixes this same issue, and I tried it out and it seemed to work 🤷 I guess the compiler is just confused. I'd suggest maybe making that change instead and avoid having to have an extra prototype.

Reply:
Good shout. Done.